README.md

```diff
@@ -105,8 +105,113 @@ Example request body of new cluster, new task definition, new service registry a
       }
     }
 }
+
+{
+    "cluster": {
+        "clustername": "myclu00",
+	"tags": [
+		{
+		    "Key": "CreatedBy",
+		    "Value": "netid"
+		},
+		{
+		    "Key": "ChargingAccount",
+		    "Value": "CX81...XXXXXXXXX.XXXXXX.XXXXXXX.YYYYYYYY..."
+		},
+		{   
+            "Key": "DRTier",
+            "Value": "3" 
+		},
+		{
+		    "Key": "OS",
+		    "Value": "container"
+		},
+		{
+		    "Key": "OwnerDepartmentContact",
+		    "Value": "netid"
+		},
+		{
+		    "Key": "spinup:spaceid",
+		    "Value": "myspace00"
+		},
+		{
+		    "Key": "OwnerDepartment",
+		    "Value": "Owner Dept"
+		},
+		{
+		    "Key": "Name",
+		    "Value": "mycontainer00"
+		},
+		{
+		    "Key": "SupportDepartment",
+		    "Value": "Support Dept"
+		},
+		{
+		    "Key": "Environment",
+		    "Value": "prd"
+		},
+		{
+		    "Key": "SupportDepartmentContact",
+		    "Value": "netid"
+		}
+	    ]
+    },
+    "taskdefinition": {
+        "family": "webservers",
+        "cpu": "256",
+        "memory": "512",
+        "containerdefinitions": [
+            {
+                "Environment": [{
+                    "Name": "API_HOST",
+                    "Value": "localhost"
+                  },{
+                    "Name": "API_PORT",
+                    "Value": "1234"
+                  }],
+                "Image": "dkw00/createsecrets:nginxenv",
+                "LogConfiguration": {
+                  "LogDriver": "awslogs",
+                  "Options": {
+                    "awslogs-group": "myclu",
+                    "awslogs-stream-prefix": "mycontainer00",
+                    "awslogs-region": "us-east-1",
+                    "awslogs-create-group": "true"
+                  }
+                },
+                "Name": "you",
+                "PortMappings": [{
+                    "ContainerPort": 80,
+                    "Protocol": "tcp"
+                  }],
+                "Secrets": []
+            }
+        ]
+    },
+    "service": {
+        "desiredcount": 1,
+        "servicename": "mycontainer00"
+    },
+    "serviceregistry": {
+      "name": "mycontainer00",
+      "cluster": "mycontainer00",
+      "dnsconfig": {
+        "namespaceid": "ns-h4yfujs27uxfefyb",
+        "dnsrecords": [
+          {
+            "ttl": 30,
+            "type": "A"
+          }
+        ]
+      }
+    }
+}
```

Comment:
maybe just add some detail to the json payload that we already had?  I dont think we need more than 1.  also, just put in one or two tags instead of all of them in the example.

Reply:
cleaned up/fixed